web/testing/regression-tests/src/known-failures.js

```diff
@@ -12,13 +12,20 @@
  * Hint: searching for "FAILED TO LOAD" in logs finds the first reference to the
  * failure for any given keyboard.
  */
+
 const knownFailures = {
-  'fv_gwichin': { /*, knownGoodCompilerVersion: '12.0.333.0', knownGoodEngineVersion: '12.0.444'*/ reason: 'Chrome ladder bug #183?' },
-  'fv_han': { reason: 'Chrome ladder bug #183?' }, 
-  'fv_northern_tutchone': { reason: 'Chrome ladder bug #183?' },
-  'fv_southern_tutchone': { reason: 'Chrome ladder bug #183?' }, 
-  'fv_tagizi_dene': { reason: 'Chrome ladder bug #183?' }, 
+  'clavbur9': { reason: 'KCCM generated but not supported #1711' },
+  'fv_gwichin': { /*, knownGoodCompilerVersion: '12.0.333.0', knownGoodEngineVersion: '12.0.444'*/ reason: 'Chrome ladder bug #183' },
+  'fv_han': { reason: 'Chrome ladder bug #183' }, 
+  'fv_northern_tutchone': { reason: 'Chrome ladder bug #183' },
+  'fv_southern_tutchone': { reason: 'Chrome ladder bug #183' }, 
+  'fv_tagizi_dene': { reason: 'Chrome ladder bug #183' }, 
   'hieroglyphic': { reason: 'Chrome ladder bug #183' }, 
```

Comment:
I see these are still listed as `knownFailures`.  Is this because the old versions would break, and so generate differences with the current versions?

- [ ] What's the plan/timeline on unlisting these?

Reply:
The plan is to remove these from the `knownFailures` array when I cherry-pick the compiler fix to stable. Then they will build correctly with the latest known stable that the regression test pulls in.